chrome/content/zotero/zoteroPane.xhtml

```diff
@@ -1334,7 +1334,7 @@
 												</toolbarbutton>
 												
 												<spacer flex="1"/>
-												<image id="zotero-tb-search-spinner" class="zotero-spinner-16"/>
+												<image id="zotero-tb-search-spinner" class="zotero-spinner-16" style="display: inline;visibility: hidden;"/>
```

Comment:
Can just move this to _quickSearchTextbox.scss, right? We're already referencing this there.

Reply:
Oh yeah, totally! Somehow I missed that `zotero-tb-search-spinner` was already in `_quickSearchTextbox.scss`.